[2022-11-29T05:28:43.397+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.task_7 manual__2022-11-29T05:28:12.931793+00:00 [queued]>
[2022-11-29T05:28:43.414+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Can_I_change_name_to_this.task_7 manual__2022-11-29T05:28:12.931793+00:00 [queued]>
[2022-11-29T05:28:43.415+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T05:28:43.416+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-29T05:28:43.416+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-29T05:28:43.434+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): task_7> on 2022-11-29 05:28:12.931793+00:00
[2022-11-29T05:28:43.444+0000] {standard_task_runner.py:55} INFO - Started process 530 to run task
[2022-11-29T05:28:43.451+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Can_I_change_name_to_this', 'task_7', 'manual__2022-11-29T05:28:12.931793+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmp2nu9qs7k']
[2022-11-29T05:28:43.453+0000] {standard_task_runner.py:83} INFO - Job 28: Subtask task_7
[2022-11-29T05:28:43.525+0000] {task_command.py:376} INFO - Running <TaskInstance: Can_I_change_name_to_this.task_7 manual__2022-11-29T05:28:12.931793+00:00 [running]> on host 68e922a5ec23
[2022-11-29T05:28:43.593+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Can_I_change_name_to_this
AIRFLOW_CTX_TASK_ID=task_7
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T05:28:12.931793+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T05:28:12.931793+00:00
[2022-11-29T05:28:43.596+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-29T05:28:43.597+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/tasks/data_cleansing_counter_stat.py']
[2022-11-29T05:28:43.617+0000] {subprocess.py:86} INFO - Output:
[2022-11-29T05:28:48.754+0000] {subprocess.py:93} INFO - 2022-11-29 05:28:48,747 | __main__ |data_cleansing_counter_stat.py | INFO | successfully put lol_basic_stat to pg and redshift
[2022-11-29T05:28:48.759+0000] {subprocess.py:93} INFO - 2022-11-29 05:28:48,756 | __main__ |data_cleansing_counter_stat.py | INFO | this task have used 2.8981580429999667s
[2022-11-29T05:28:49.063+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-29T05:28:49.144+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Can_I_change_name_to_this, task_id=task_7, execution_date=20221129T052812, start_date=20221129T052843, end_date=20221129T052849
[2022-11-29T05:28:49.256+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-29T05:28:49.295+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
